config/locales/zh-CN.yml

```diff
@@ -92,7 +92,7 @@ zh-CN:
         uptime: 服务状况
       header:
         dashboard: 控制台
-        search_gem: 搜索 Gems&hellip;
```

Comment:
I prefer the original one here. See https://github.com/rubygems/rubygems.org/pull/1803#issuecomment-431605597